src/test/components/__snapshots__/TooltipMarker.test.js.snap

```diff
@@ -3306,13 +3306,31 @@ exports[`TooltipMarker shows a tooltip for Jank markers 1`] = `
       <div
         class="tooltipTitle"
       >
-        70.00ms event processing delay
+        Jank – event processing delay
       </div>
     </div>
   </div>
   <div
     class="tooltipDetails"
   >
+    <div
+      class="tooltipLabel"
+    >
+      Delay
+      :
+    </div>
+    70ms
```

Comment:
This is orthogonal to this patch, so no change needed: I wonder if this isn't duplicated informtion, as this value is the same than the marker's duration.

Reply:
I'm going to fix this, because you are absolutely right. It's simpler without adding this additional duplicate information.